packages/tldraw/src/index.ts

```diff
@@ -494,9 +494,12 @@ export {
 } from './lib/ui/context/toasts'
 export { useCanRedo, useCanUndo } from './lib/ui/hooks/menu-hooks'
 export { useMenuClipboardEvents, useNativeClipboardEvents } from './lib/ui/hooks/useClipboardEvents'
+export {
+	useCollaborationStatus,
+	useShowCollaborationUi,
+} from './lib/ui/hooks/useCollaborationStatus'
```

Comment:
could we keep the old name as an alias but mark it as `@deprecated` so this isn't a breaking change?

Also, slightly surprised that there wasn't an `api-report` change generated from this

Reply:
omg i actually can't read pls ignore